pep-0687.rst

```diff
@@ -2,14 +2,15 @@ PEP: 687
 Title: Isolating modules in the standard library
 Author: Erlend Egeberg Aasland <erlend.aasland@protonmail.com>, Petr Viktorin <encukou@gmail.com>
 Discussions-To: https://discuss.python.org/t/14824
-Status: Draft
+Status: Accepted
 Type: Standards Track
 Content-Type: text/x-rst
 Requires: 489, 573, 630
 Created: 04-Apr-2022
 Python-Version: 3.12
 Post-History: `04-Apr-2022 <https://discuss.python.org/t/14824>`__,
               `11-Apr-2022 <https://mail.python.org/archives/list/python-dev@python.org/thread/B3HYQIE4Z5WBJCC3FUZJZHXLM32I4BZA/>`__
+Resolution: https://discuss.python.org/t/14824
```

Comment:
```suggestion
Resolution: https://discuss.python.org/t/14824/4
```

Better to link to the specific message approving the PEP, rather than just the top of the discussion thread